### ISSUE TITLE
Incorrect version downloaded for all browsers on Windows

### ISSUE THREAD

**Issue by scjupalli** [bonigarcia/webdrivermanager#351]
**Description of the problem**: When trying to setup the chrome or firefox driver on windows 7 and 10 machines, it's always downloading the latest driver irrespective of the browser version currently installed. On further investigation I found that it was always looking in the wrong folder

**Browser and version**: 
Chrome 74 and 75 (both 64 bit)
Mozilla - 60 (ESR) and 67 (32 & 64)

**Operating system**:
Windows 7 & 10
architecture - 32 and 64 bits

**WebDriverManager version**: 3.6.1

**WebDriverManager use**:
```java
switch (browserName) {
        case "CHROME":
                System.out.println("Setting up Chrome Local Browser");
                WebDriverManager.chromedriver().setup();
                System.out.println("Downloaded Chrome version --> " + WebDriverManager.chromedriver().getDownloadedVersion());
                driver = new ChromeDriver();
                break;
        default:
                System.out.println("Setting up Local Browser(FireFox)");
                WebDriverManager.firefoxdriver().setup();
                System.out.println("Downloaded Firefox version --> " + WebDriverManager.firefoxdriver().getDownloadedVersion());
                driver = new FirefoxDriver();
}
return driver;
```

**Probable Cause**: 
If the OS Architecture is 64 we are asking to look under "Program Files (x86)" folder instead of "Program Files".

```Java
    protected String getProgramFilesEnv() {
        return System.getProperty("os.arch").contains("64")
                ? "PROGRAMFILES(X86)"
                : "PROGRAMFILES";
    }
```

**Recommendation/Suggestion**: 
If [getDefaultBrowserVersion](https://github.com/bonigarcia/webdrivermanager/blob/master/src/main/java/io/github/bonigarcia/wdm/WebDriverManager.java#L1138) method in [WebDriverManager.java](https://github.com/bonigarcia/webdrivermanager/blob/master/src/main/java/io/github/bonigarcia/wdm/WebDriverManager.java) could handle below logic that would be great, any thoughts?

we can have 32-bit version of the applications installed on 64-bit architecture, hence I think we need to check 32-bit installation when nothing is found under 64-bit

![image](https://user-images.githubusercontent.com/13438846/59704080-501c5980-91c9-11e9-8523-f761f0613916.png)

**Comment by bonigarcia** [comment]
@scjupalli Thanks a lot for the complete description. The suggested solution looks nice, I am going to look into it.

**Comment by bonigarcia** [comment]
A similar solution has been implemented in release 3.6.2, just released.
